Formula/gnupg.rb

```diff
@@ -36,13 +36,18 @@ class Gnupg < Formula
   end
 
   def install
+    # Fixed upstream: https://dev.gnupg.org/rGd36c4dc95b72b780375d57311bdf4ae842fd54fa
+    # remove in the next release
+    inreplace "tests/openpgp/defs.scm",
+              "#use-keyboxd\")",
+              "#use-keyboxd\")\n(string-append \"keyboxd-program \" (tool 'keyboxd))"
```

Comment:
Let's use an embedded patch for this (as in with a `patch :DATA` stanza). An outdated patch is easier to spot, because CI will fail when it fails to apply.

Reply:
Sure. Currently running install locally to make sure correct patch.